compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -367,6 +367,36 @@ class TransposeTestGraph final : public luci::test::TestIOGraph
   luci::CircleConst *_perm = nullptr;
 };
 
+class ConcatenationTestGraph final : public luci::test::TestIOGraph
+{
+public:
+  void init(void)
+  {
+    TestIOGraph::init({16}, {32});
+    _param = g()->nodes()->create<luci::CircleConst>();
```

Comment:
maybe use `create_dummy_const()` ?

Reply:
That seems style change. I'll update it with other graphs as well.